web/models/RespondentStatsMetrics.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Prometheus.RespondentStatsMetrics do
+  use Prometheus.Collector
+
+  alias Ask.Repo
+
+  def collect_mf(_registry, callback) do
+    respondent_stats_query = "select rs.* from respondent_stats rs"
+    query_results = Repo.query!(respondent_stats_query)
+    columns = query_results.columns|> Enum.map(fn(column_name) -> String.to_charlist(column_name) end)
+    rows = query_results.rows |> Enum.map(fn(row) -> formatRow(row) end)
+    result_list =  Enum.map(rows, fn row ->
+                                      columns
+                                      |> Enum.zip(row)
+                                      |> Enum.into([])
+                            end)
+    callback.(create_gauge(
```

Comment:
This is not generating the gauge with the right shape. The `count` value from the table should be used as the value for each set of labels.

Also I'd not "reflect" on the columns. The `mode` column currently stores a JSON with an array of strings. Let's convert that into a comma separated string for the `mode` label in the gauge.